rllib/core/rl_trainer/tests/test_trainer_runner.py

```diff
@@ -2,55 +2,130 @@
 import unittest
 import ray
 import time
+import numpy as np
+import itertools
 
 from ray.rllib.policy.sample_batch import DEFAULT_POLICY_ID, MultiAgentBatch
-from ray.rllib.utils.test_utils import get_cartpole_dataset_reader
+from ray.rllib.utils.test_utils import check, get_cartpole_dataset_reader
+from ray.rllib.utils.framework import try_import_tf
+from ray.rllib.core.rl_trainer.scaling_config import TrainerScalingConfig
 from ray.rllib.core.testing.utils import (
     get_trainer_runner,
+    get_rl_trainer,
     add_module_to_runner_or_trainer,
 )
 
 
 class TestTrainerRunner(unittest.TestCase):
-    """This test is setup for 2 gpus."""
-
-    # TODO: This unittest should also test other resource allocations like multi-cpu,
-    # multi-node multi-gpu, etc.
-
     @classmethod
-    def setUp(cls) -> None:
+    def setUpClass(cls) -> None:
         ray.init()
 
+        # Settings to test
+        cls.scaling_configs = {
```

Comment:
it's not a big deal. just making this a class variable feels a bit unnecessary.
can be a GLOBAL_CONSTANT etc.

Reply:
I am neutral to this. Leaving it as is, cuz I don't wanna resubmit the PR :) . Let's just go with this one since all the tests are passing.